packages/snowpack/src/commands/build-util.ts

```diff
@@ -1,305 +0,0 @@
-    throw new Error('Imports in CSS Modules are not yet supported.');
-  });
-  return `${
-    hmr
-      ? `
-import * as __SNOWPACK_HMR_API__ from '${getMetaUrlPath('hmr.js', isDev, config)}';
-import.meta.hot = __SNOWPACK_HMR_API__.createHotContext(import.meta.url);
-import.meta.hot.accept(({module}) => {
-  code = module.code;
-  json = module.default;
-});
-import.meta.hot.dispose(() => {
-  document.head.removeChild(styleEl);
-});\n`
-      : ``
-  }
-export let code = ${JSON.stringify(injectableSource)};
-let json = ${JSON.stringify(exportTokens)};
-export default json;
-
-const styleEl = document.createElement("style");
-const codeEl = document.createTextNode(code);
-styleEl.type = 'text/css';
-
-styleEl.appendChild(codeEl);
-document.head.appendChild(styleEl);`;
-}
-
-export function wrapHtmlResponse({
-  code,
-  isDev,
-  hmr,
-  config,
-}: {
-  code: string;
-  isDev: boolean;
-  hmr: boolean;
-  config: SnowpackConfig;
-}) {
-  // replace %PUBLIC_URL% in HTML files (along with surrounding slashes, if any)
-  code = code.replace(/\/?%PUBLIC_URL%\/?/g, config.buildOptions.baseUrl);
-
-  if (hmr) {
-    code += `<script type="module" src="${getMetaUrlPath('hmr.js', isDev, config)}"></script>`;
-  }
-  return code;
-}
-
-export function wrapEsmProxyResponse({
-  url,
-  code,
-  ext,
-  isDev,
-  hmr,
-  config,
-}: {
-  url: string;
-  code: string;
-  ext: string;
-  isDev: boolean;
-  hmr: boolean;
-  config: SnowpackConfig;
-}) {
-  if (ext === '.json') {
-    return `${
-      hmr
-        ? `
-    import * as __SNOWPACK_HMR_API__ from '${getMetaUrlPath('hmr.js', isDev, config)}';
-    import.meta.hot = __SNOWPACK_HMR_API__.createHotContext(import.meta.url);
-    import.meta.hot.accept(({module}) => {
-      json = module.default;
-    });`
-        : ''
-    }
-let json = ${JSON.stringify(JSON.parse(code))};
-export default json;`;
-  }
-
-  if (ext === '.css') {
-    return `${
-      hmr
-        ? `
-import * as __SNOWPACK_HMR_API__ from '${getMetaUrlPath('hmr.js', isDev, config)}';
-import.meta.hot = __SNOWPACK_HMR_API__.createHotContext(import.meta.url);
-import.meta.hot.accept();
-import.meta.hot.dispose(() => {
-  document.head.removeChild(styleEl);
-});\n`
-        : ''
-    }
-const code = ${JSON.stringify(code)};
-
-const styleEl = document.createElement("style");
-const codeEl = document.createTextNode(code);
-styleEl.type = 'text/css';
-
-styleEl.appendChild(codeEl);
-document.head.appendChild(styleEl);`;
-  }
-
-  return `export default ${JSON.stringify(url)};`;
-}
-
-const PUBLIC_ENV_REGEX = /^SNOWPACK_PUBLIC_/;
-export function generateEnvModule(mode: 'development' | 'production') {
-  const envObject = {...process.env};
-  for (const env of Object.keys(envObject)) {
-    if (!PUBLIC_ENV_REGEX.test(env)) {
-      delete envObject[env];
-    }
-  }
-  envObject.MODE = mode;
-  envObject.NODE_ENV = mode;
-  return `export default ${JSON.stringify(envObject)};`;
-}
```

Comment:
I like the new structure. I think this file started to get a little “junk drawer”-ish over time.

Reply:
yea definitely. It's funny, but only in this PR did the two concepts we got going on here finally jump out to me.